custom/ilsgateway/zipline/helpers.py

```diff
@@ -0,0 +1,51 @@
+from pytz import timezone
+
+
+def format_date(datetime):
+    return timezone('Africa/Dar_es_Salaam').localize(datetime).strftime('%Y-%m-%d %H:%M:%S')
+
+
+def status_date_or_empty_string(status):
+    if status:
+        return format_date(status.timestamp)
```

Comment:
see the spec - sometimes we need to use timestamp, sometimes we need to use zipline_timestamp